ui/js/models/resultset.js

```diff
@@ -191,9 +191,33 @@ treeherder.factory('ThResultSetModel', ['$rootScope', '$http', '$location',
                 return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri);
             },
 
-            triggerMissingJobs: function (resultset_id, repoName) {
+            triggerMissingJobs: function (resultset_id, repoName, decisionTaskID) {
                 var uri = resultset_id + '/trigger_missing_jobs/';
-                return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri);
+                return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri).then(function () {
```

Comment:
It looks like this tries the "old way" first?  Assuming this POST is the old way..